packages/synapse-react-client/src/synapse-queries/trash/useTrashCan.ts

```diff
@@ -54,7 +53,7 @@ export function useRestoreEntities(
     UseMutationOptions<
       PromiseSettledResult<void>[],
       SynapseClientError,
-      string | Set<string>
+      string | string[] | Set<string>
```

Comment:
TanStack table uses `string[]` instead of `Set<string>` to capture selection, so update the allowed types so we can just pass the array.